packages/invoices/src/OutboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -133,24 +139,49 @@ const useStatusChangeButton = () => {
     setSelectedOption(() => getNextStatusOption(status, options));
   }, [status, options]);
 
-  return { options, selectedOption, setSelectedOption, onGetConfirmation };
+  return { options, selectedOption, setSelectedOption, getConfirmation };
+};
+
+const useStatusChangePlaceholderCheck = () => {
```

Comment:
Aaaand the actual change for the issue. Ended up being so simple and I think is informative for the user rather than just disabling the status? 🤔

Reply:
much better than disabling - which could be confused with  being disabled because on hold, or finalised.
🤔  on hold doesn't disable anything.. should it?